lib/src/value/boolean.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {hash} from 'immutable';
+
+import {Value} from './value';
+
+/**
+ * Sass boolean.
+ *
+ * Cannot be constructed; exists only as an interface and the exported
+ * singletons.
+ */
+export interface SassBoolean extends Value {
+  value: boolean;
+}
+
+const trueHash = hash(true);
+const falseHash = hash(true);
+
+class SassBooleanInternal extends Value implements SassBoolean {
```

Comment:
Can we call this just `SassBoolean`, like we do for `SassNull`?

Reply:
Unfortunately no; the interface and class names clash.